packages/botonic-plugin-hubtype-analytics/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "include": ["src/"],
+  "compilerOptions": {
+    "jsx": "react",
+    "experimentalDecorators": true,
+    "baseUrl": ".",
+    "outDir": "./lib/cjs",
+    "esModuleInterop": true,
+    "lib": ["es2015"],
+    "skipLibCheck": true
```

Comment:
`skipLibCheck``is a pending assignature I have to investigate, I think the ideal would be to have this to false. Let’s keep iterating though!